relay-server/src/endpoints/common.rs

```diff
@@ -67,6 +67,9 @@ pub enum BadStoreRequest {
     #[error("missing minidump")]
     MissingMinidump,
 
+    #[error("invalid compression container")]
+    InvalidCompressionContainer(#[from] std::io::Error),
```

Comment:
I would not make this a `From` impl. `BadStoreRequest: From<std::io::Error>` implies that there is one obvious way to turn an `std::io::Error` into a `BadStoreRequest`, but that isn't the case: there is also the `InvalidBody` variant.

```suggestion
    InvalidCompressionContainer(#[source] std::io::Error),
```

Reply:
Thanks. What an easy mistake to introduce.